ui/eureka/eureka.js

```diff
@@ -1732,237 +1732,446 @@ const Options = {
           label: {
-            en: 'Sneak',
+            en: 'Sneak & Spell',
             de: 'Taktisches Gemetzel',
             fr: 'Les yeux de l\'ennemi',
             ja: '術士大隊との会敵',
             cn: '遭遇术师大队',
             ko: '술사대대 발견',
           },
+          dropsFieldNotes: false,
+          shortLabel: {
+            en: 'Sneak',
+            fr: 'Yeux',
+          },
           x: 20.3,
           y: 26.8,
           fateID: 1597,
         },
         robots: {
           label: {
-            en: 'Robots',
+            en: 'None of Them Knew They Were Robots',
             de: 'Nichts als Schrott',
             fr: 'Les araignées de fer',
             ja: '無人魔導兵器との会敵',
             cn: '遭遇无人魔导兵器',
             ko: '무인 마도 병기 발견',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 8,
+          shortLabel: {
+            en: 'Robots',
+            fr: 'Araignées',
+          },
           x: 24.8,
           y: 27.5,
           fateID: 1598,
         },
         beasts: {
           label: {
-            en: 'Beasts',
+            en: 'The Beasts Must Die',
             de: 'Husch, ins Körbchen!',
             fr: 'Museler le Chien',
             ja: '忠犬との遭遇',
             cn: '发现忠犬',
             ko: '충견과 조우하다',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 3,
+          shortLabel: {
+            en: 'Beasts',
+            fr: 'Museler',
+          },
           x: 20.3,
           y: 26.8,
           fateID: 1599,
         },
         unrest: {
           label: {
-            en: 'Unrest',
+            en: 'Unrest for the Wicked',
             de: 'Wer rastet, der blutet',
             fr: 'Pas de quartier',
             ja: '術士大隊への奇襲',
             cn: '奇袭术师大队',
             ko: '술사대대 기습',
           },
+          dropsFieldNotes: false,
+          shortLabel: {
+            en: 'Unrest',
+            fr: 'Pas de quartier',
+          },
           x: 24.8,
           y: 27.5,
           fateID: 1600,
         },
         machine: {
           label: {
-            en: 'Machine',
+            en: 'More Machine Now Than Man',
             de: 'Auf zum Gegenangriff',
             fr: 'Machines aux trousses',
             ja: '有人魔導兵器の迎撃',
             cn: '迎击有人魔导兵器',
             ko: '유인 마도 병기 요격',
           },
+          isCEPrecursor: true,
+          dropsFieldNotes: true,
+          fieldNotes: 1,
+          shortLabel: {
+            en: 'Machine',
+            fr: 'Machine',
+          },
           x: 28.4,
           y: 29.3,
           fateID: 1601,
         },
         plants: {
           label: {
-            en: 'Plants',
+            en: 'Can Carnivorous Plants Bloom Even on a Battlefield?',
             de: 'Linientreue',
-            fr: 'Des racines et des crocs',
+            fr: 'Des Racines et des Crocs',
             ja: '野生生物を排除せよ',
             cn: '排除野生生物',
             ko: '야생 생물을 제거하라',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 5,
```

Comment:
`dropsFieldNotes` seems a little redundant here, as you could just have `fieldNotes` and determine `dropsFieldsNotes` from whether `fieldNotes` was set, I think?

Reply:
I added the fieldNotes entry later actually so feel free to remove it if you consider it better ?

Reply:
Yeah, I think removing that field sounds good.  Would you mind doing that?

Reply:
done !